frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/AppearanceSettings.tsx

```diff
@@ -48,7 +48,10 @@ export const AppearanceSettings = ({
     }),
   );
   const upgradePageUrl = useSelector(state =>
-    getUpgradeUrl(state, { utm_media: "static-embed-settings-appearance" }),
+    getUpgradeUrl(state, {
```

Comment:
there was a conflict in this PR because `LookAndFeelSettings` isn't in v50.

I ported [the whole change](https://github.com/metabase/metabase/pull/47116/files#diff-5e616c86171e3067ae70f5bc940f8eba07c534241d9231b05698e0581d05319b), but changed the `utm_content` to `static-embed-settings-appearance` to reflect the current version tab name.

Reply:
I changed it back to `static-embed-settings-look-and-feel` [as decided.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1724419214125029?thread_ts=1724077872.663639&cid=C063Q3F1HPF)